inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -66,6 +69,210 @@ namespace details {
         return nullptr;\
     });\
 
+/// \brief Creates legacy representation of CNNLayer for SubGraphOp.
+/// \param layer node type
+/// \return pointer to CNNLayer with legacy representation of SubGraphOp.
+CNNLayer::Ptr createSubGraphLayer(const std::shared_ptr<ngraph::Node>& layer) {
+    auto sub_graph = std::dynamic_pointer_cast<ngraph::op::util::SubGraphOp>(layer);
+    if (!sub_graph) {
+        THROW_IE_EXCEPTION << "Cannot cast layer to SubGraphOp.";
+    }
+
+    // inputs/outputs of TensorIterator (ngraph representation)
+    auto parameters = sub_graph->get_function()->get_parameters();
+    auto results = sub_graph->get_function()->get_results();
```

Comment:
Is it a legacy code?
@itikhono should we also get sinks from a function?

Reply:
This is a copy of legacy function createSubGraphLayer().